.github/actions/set-git-config/action.yml

```diff
@@ -7,5 +7,5 @@ runs:
     - name: Set git config
       shell: bash
       run: |
-        git config --global user.email "jackblakeston93@gmail.com"
-        git config --global user.name "Jack Blakeston"
+        git config --global user.email "no.gravity.corporation@gmail.com"
```

Comment:
should we add the email and username to the secrets? I don't like having the email here

Reply:
Sure, good idea. Let's go over some details though